### PR TITLE
Tiers decoupling wire up

### PR DESCRIPTION
**Pull request by allouis**
Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [ ] There's a clear use-case for this code change, explained below
- [ ] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

Also, if you'd be interested in writing code like this for us more regularly, we're hiring:
https://careers.ghost.org

### CODE REVIEW

File: ghost/core/core/server/api/endpoints/tiers.js [hunk 1]
```diff
@@ -47,26 +29,11 @@ module.exports = {
         data: [
             'id'
         ],
-        validation: {
```
Comment (daniellockyer) [inline ghost/core/core/server/api/endpoints/tiers.js:50]: GitHub won't let me comment on the above, but we can delete the following now too:

```
options: [ 
    'include'
],
```

File: ghost/core/core/server/api/endpoints/tiers.js [hunk 2]
```diff
@@ -100,23 +63,11 @@ module.exports = {
         headers: {
             cacheInvalidate: true
         },
-        validation: {
```
Comment (daniellockyer) [inline ghost/core/core/server/api/endpoints/tiers.js:103]: Should we be keeping `id` as an `option` here? Or is it actually optional now?

Comment (allouis) [reply ghost/core/core/server/api/endpoints/tiers.js:103]: No that's my B - will fix

File: ghost/payments/lib/payments.js [hunk 3]
```diff
@@ -12,13 +14,237 @@ class PaymentsService {
...
+        /** @private */
         this.offersAPI = deps.offersAPI;
         /** @private */
         this.stripeAPIService = deps.stripeAPIService;
 
         DomainEvents.subscribe(OfferCreatedEvent, async (event) => {
             await this.getCouponForOffer(event.data.offer.id);
         });
+
+        DomainEvents.subscribe(TierCreatedEvent, async (event) => {
```
Comment (daniellockyer) [inline ghost/payments/lib/payments.js:31]: This is pretty cool now

**Pull request closed without merging**